aiosmtpd/smtp.py

```diff
@@ -119,23 +147,27 @@ class SMTP(asyncio.StreamReaderProtocol):
     # (RFC 5322 s 2.1.1 + RFC 6532 s 3.4) 998 octets + CRLF = 1000 octets
     # (RFC 5321 s 4.5.3.1.6) 1000 octets + "transparent dot" = 1001 octets
 
-    def __init__(
-            self, handler,
-            *,
-            data_size_limit=DATA_SIZE_DEFAULT,
-            enable_SMTPUTF8=False,
-            decode_data=False,
-            hostname=None,
-            ident=None,
-            tls_context: Optional[ssl.SSLContext] = None,
-            require_starttls=False,
-            timeout=300,
-            auth_required=False,
-            auth_require_tls=True,
-            auth_exclude_mechanism: Optional[Iterable[str]] = None,
-            auth_callback: Callable[[str, bytes, bytes], bool] = None,
-            loop=None,
-    ):
+    # base64-encoded 'User Name\x00'
+    AuthLoginUsernameChallenge = "VXNlciBOYW1lAA=="
+    # base64-encoded 'Password\x00'
+    AuthLoginPasswordChallenge = "UGFzc3dvcmQA"
```

Comment:
Some noncompliant SMTP client implementation expects "Username:" and "Password:". To cater for those noncompliant clients, the AUTH LOGIN challenges are customizable.

(Standard says that clients should totally ignore the challenges.)